core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -21,6 +21,7 @@ import java.io.{BufferedInputStream, FileNotFoundException, IOException, InputSt
 import java.util.concurrent.{ExecutorService, Executors, TimeUnit}
 
 import scala.collection.mutable
+import scala.collection.mutable.ArrayBuffer
```

Comment:
nit: you could say `new mutable.ArrayBuffer` to avoid the new import. That also makes it consistent with the rest of the file.